source/administration/config-settings.rst

```diff
@@ -6,41 +6,20 @@ The first user added to a new Mattermost install is assigned the System Admin ro
 
 Note: For any setting not explicitly set in ``config.json`` the Mattermost server will use the default value as documented here, which can be observed in the default ``config/config.json`` file included in each Mattermost release.
 
-Quick Links:
-
-`General`_
-	`Configuration`_ - `Localization`_ - `Users and Teams`_ - `Privacy`_ - `Policy`_ - `Compliance`_ - `Logging`_
-
-`Authentication`_
-	`Email Auth`_ - `OAuth 2.0`_ - `GitLab`_ - `Google`_ - `Office 365`_ - `AD/LDAP`_ - `SAML`_ - `MFA`_
-
-`Security`_
-	`Sign Up`_ - `Password`_ - `Public Links`_ - `Sessions`_ - `Connections`_
-
-`Notifications`_
-	`Email`_ - `Mobile Push`_
-
-`Integrations`_
-	`Custom Integrations`_ - `WebRTC (Beta)`_ - `External Services`_
-
-`Files`_
-	`Storage`_ - `Images`_
-
-`Customization`_
-	`Custom Branding`_ - `Custom Emoji`_ - `Legal and Support`_ - `Mattermost App Links`_
-
-`Advanced`_
-	`Rate Limiting`_ - `Database`_ - `Developer`_ - `High Availability (Beta)`_ - `Performance Monitoring (Beta)`_
+.. contents::
```

Comment:
For some reason this isnt rendering in GitHub, propose we do a post merge verification